### DISCUSSION TITLE
Could / should the default Address element's countryCode be set based on locale or configurable?

### DISCUSSION THREAD

**Discussion by samhibberd** [craftcms/cms#13478] [Ideas] [answered]
Whilst getting to grips with Craft 4 new addressing logic, we noticed that the default countryCode (that determines the available fields) for new addresses is the US, this is both in the cp and it if creating an address programmatically. 

Figured it would be nice if it was set to the site / user lang / locale or was perhaps configurable via `app.php`.

**Comment by timprint** [comment]
Another request for this.

I'm not sure what the best solution is but having some way to set the default countryCode would definitely be welcome. I am currently building a 100% UK based site and currently all new addresses default to US. It isn't a great user experience. Even null would be a better option than US, that really doesn't make any sense in this context.

**Comment by brandonkelly** [comment, marked as answer]
Added a new `defaultCountryCode` config setting for Craft 4.5 (109beec325c7ec3586ad3b7b6c5f0c221604a60f). Thanks for the suggestion!

**Reply by brandonkelly** [reply]
Craft 4.5.0 is out now with that setting.
